recipe/meta.yaml

```
@@ -8,30 +8,30 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/polyagamma-{{ version }}.tar.gz
-  sha256: 84d7527f26d1e376996c0a6fc1c603db306fb70a079fe5091824c7be04625731
+  sha256: 0b823b4b386fc7ac5e1a1829b123eaefe881bce078240693c209a30b9da3d7b2
 
 build:
-  number: 1
+  number: 0
   script: {{ PYTHON }} setup.py build_ext --inplace && {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<39]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - setuptools >=61.0.0
-    - setuptools-scm
+    - numpy ==1.19.3  # [py==39]
+    - numpy ==1.21.3  # [py==310]
+    - numpy ==1.23.2  # [py==311]
+    - numpy ==1.26.0  # [py==312]
```

Keep this plain and rerender, conda-forge has global pinnings for this that match what you want.
```suggestion
    - numpy
```